common/determined_common/api/experiment.py

```diff
@@ -25,48 +27,48 @@ def activate_experiment(master_url: str, exp_id: int) -> None:
 
 @authentication_required
 def logs(args: Namespace) -> None:
-    last_offset, last_state = 0, None
-
-    def print_logs(offset: Optional[int], limit: Optional[int] = 5000) -> Any:
-        nonlocal last_offset, last_state
-        path = "trials/{}/logsv2?".format(args.trial_id)
+    def print_logs(
+        offset: Optional[int], limit: Optional[int] = 5000, follow: bool = False
+    ) -> None:
+        path = "/api/v1/trials/{}/logs?".format(args.trial_id)
+        query = {}  # type: Dict[str, Any]
         if offset is not None:
-            path += "&offset={}".format(offset)
+            query["offset"] = offset
         if limit is not None:
-            path += "&limit={}".format(limit)
-        logs = api.get(args.master, path).json()
-        for log in logs:
-            print(log["message"], end="")
-            last_state = log["state"]
-        return logs[-1]["id"] if logs else last_offset
+            query["limit"] = limit
+        if follow:
+            query["follow"] = "true"
+        for f in [
+            "agent_id",
+            "container_id",
```

Comment:
blocking: Needs to match the new `dest`s:
```suggestion
            "agent_ids",
            "container_ids",
```
Although, now that I think about it, we should probably actually go back and put everything in singular, including in the proto definitions. No reason to have `agent_ids` but `level`, etc. Sorry for going back and forth on this one.

Reply:
> Although, now that I think about it, we should probably actually go back and put everything in singular, including in the proto definitions.

The weird part about this is, then, the internal repr ends up being a field in go called `AgentId` that is `[]int32`, which is strange. How about I make them all plural?

Reply:
I went with plural, like it much more. Lmk what you think.